### PR TITLE
ALPHA-498 reindex collections

### PR DESCRIPTION
**Pull request by atapin**

### CODE REVIEW

**Comment by augur** [comment]
Tests failed on Jenkins

**Comment by augur** [comment]
I can't find where this code is launched from

**Comment by augur** [comment]
Consider targeting this to 1.27

File: worker/src/main/kotlin/com/rarible/protocol/union/worker/task/search/ReindexService.kt [hunk 1]
```diff
@@ -37,6 +43,9 @@ class ReindexService(
             EsEntity.ITEM -> {
                 scheduleItemReindex(newIndexName)
             }
+            EsEntity.COLLECTION -> {
+                scheduleCollectionReindex(newIndexName)
+            }
             EsEntity.ORDER,
                 // EsEntity.COLLECTION,
```
Comment (augur) [inline worker/src/main/kotlin/com/rarible/protocol/union/worker/task/search/ReindexService.kt:50]: Remove this please

File: worker/src/main/kotlin/com/rarible/protocol/union/worker/task/search/collection/CollectionTaskParam.kt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package com.rarible.protocol.union.worker.task.search.collection
+
+import com.rarible.protocol.union.dto.BlockchainDto
+
+class CollectionTaskParam(
```
Comment (zolt) [inline worker/src/main/kotlin/com/rarible/protocol/union/worker/task/search/collection/CollectionTaskParam.kt:5]: looks like data class

**Review by zolt — CHANGES_REQUESTED** [review]

**Review by zolt — APPROVED** [review]

**Pull request merged**